src/interactive.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-cycle */
+import moreIcon from './more.svg';
+import { todoList, todoTasks, elementGenerator } from './index.js';
+
+function savedList() {
+  localStorage.setItem('ToDo', JSON.stringify(todoTasks));
+}
+
+function listItem(elem) {
+  const list = elementGenerator('li', 'task draggable', null, null);
+  const flex = elementGenerator('div', 'flex', null, null);
+  const oneTodo = elementGenerator('input', 'one-todo', null, null);
+  oneTodo.type = 'checkbox';
+  oneTodo.checked = elem.checked;
+
+  const form = elementGenerator('form', 'edit', null, null);
+  const input = elementGenerator('input', 'label', null, null);
+  input.value = elem.description;
+  const image = elementGenerator('img', 'more', null, null);
+  image.src = moreIcon;
+
+  if (elem.checked) {
+    input.classList.add('line-through');
+  } else {
+    input.classList.remove('line-through');
+  }
```

Comment:
- [ ] Take out this status check on another file, the new file should contain methods related to the status updates (completed: true / false)

Reply:
Thank you, I have now made the required changes